web/themes/custom/cloudy/templates/content/page/node--page--full.html.twig

```diff
@@ -55,5 +55,23 @@
     {% if view_past_events > 0 %}
       {{ drupal_block('views_block:events_on_content-block_past_events') }}
     {% endif %}
+    {% set view_construction =
+      drupal_block('views_block:construction_projects_on_content-block_active_on_content')|length
+    %}
+    {% if view_construction > 0 %}
+      {{ drupal_block('views_block:construction_projects_on_content-block_active_on_content') }}
+    {% endif %}
+    {% set view_construction =
```

Comment:
Each set should be a unique variable like "view_upcoming_construction" and "view_completed_construction".